dev/user.clj

```diff
@@ -1,153 +1,2 @@
 (ns user
-  (:require [clojure.tools.namespace.repl :as tn :refer [refresh refresh-all]]
```

Comment:
I'm not sure if you're planning on adding these later. If so, please disregard.  Otherwise, there are a lot of useful namespaces here besides `fluree.db.json-ld.api`.

Reply:
I think I'd prefer letting someone who needs to use the user.clj file fill in the namespaces they need, I'm not sure it needs to be done as part of this commit.

Reply:
Then why remove the file at all?

Reply:
It was filled with obsolete code, so it seemed simpler to start from a clean slate. I can resurrect it if you'd like.

Reply:
Maybe this cleanup is outside the scope of this PR. I'm using the `user` ns extensively right now, I can commit any useful changes. For now, maybe we can just leave it?

Reply:
I'd rather err on the side of not breaking anyone's work flow. If the code is in fact obsolete, then it's fine to remove, but most of those namespace declarations are not, and are useful in a repl session, such as all the core async namespaces, the benchmarking namespace, loging, util, etc.

I think if you're going to remove anything from the user namespace, it should be limited to code that's actually obsolete.

Reply:
Yeah, I was too hasty here. I'm resurrecting it, but I'll remove all the references to `fluree.db.api`.